Makefile

```diff
@@ -41,13 +41,13 @@ build:
 	CGO_ENABLED=0 go test ./suites/scale -v -c -o ./out/osde2e-scale
 
 test:
-	go test $(E2E_PKG) -test.v -ginkgo.skip="$(GINKGO_SKIP)" -ginkgo.focus="$(GINKGO_FOCUS)" -test.timeout 8h 
+	go test $(E2E_PKG) -test.v -ginkgo.skip="$(GINKGO_SKIP)" -ginkgo.focus="$(GINKGO_FOCUS)" -test.timeout 8h -e2e-config $(E2ECONFIG)
```

Comment:
Just confirming: this behaves as expected when no E2ECONFIG is provided?

Reply:
Yes and no. 

Yes, it will.

No it doesn't because I didn't put `=` between the flag and the string. New commit inc.